docs/reference/modules/plugins.asciidoc

```diff
@@ -14,19 +14,26 @@ and more.
 ==== Installing plugins
 
 Installing plugins can either be done manually by placing them under the
-`plugins` directory, or using the `plugin` script. Several plugins can
-be found under the https://github.com/elasticsearch[elasticsearch]
-organization in GitHub, starting with `elasticsearch-`.
+`plugins` directory, or using the `plugin` script.
 
 Installing plugins typically take the following form:
 
 [source,shell]
 -----------------------------------
+bin/plugin --install plugin_name
+-----------------------------------
+
+The plugin will be automatically downloaded in this case from `download.elastic.co` download service using the
```

Comment:
can we do this in another PR maybe (dont wanna bloat this one and make sure it gets in). Lets keep it as is for now, but I am all for https with a possibility to fallback to http (via CLI parameter).